Sources/Contour/Caching/ICacheProvider.cs

```diff
@@ -1,82 +0,0 @@
-﻿namespace Contour.Caching
-{
-    using System;
-
-    using Contour.Helpers;
-
-    /// <summary>
-    /// The CacheProvider interface.
-    /// </summary>
-    public interface ICacheProvider
```

Comment:
Breaking backward compatibility. You should avoid it.

Reply:
Reverted deletion and marked as obsolete.